biz.aQute.bnd/src/aQute/bnd/main/RemoteCommand.java

```diff
@@ -109,6 +116,120 @@ public void event(Event e) throws Exception {
 		agent = launcher.getAgent();
 	}
 
+	@Description("Communicate with the remote framework to list the installed bundles")
+	interface ListOptions extends Options {
+		@Description("Specify the type of output - currently supported option is 'json'")
+		String output();
```

Comment:
In the sake of type safety, this is better a flag: -j/--json. I.e. boolean json()

Reply:
Done